sdk/documentintelligence/Azure.AI.DocumentIntelligence/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed a bug where calling `Operation.Id` would sometimes return an `InvalidOperationException` with message "The operation ID was not present in the service response.".
```

Comment:
It's still unclear to me why this bug was not caught by tests in playback mode. Live testing fails deterministically. I wonder if there was any change to the service's behavior.